lib/shelley/test/data/cardano-node-shelley/shelley-genesis.yaml

```diff
@@ -47,12 +47,12 @@ maxLovelaceSupply: 1000000000000000000
 protocolMagicId: 764824073
 networkMagic: 764824073
 networkId: Mainnet
-epochLength: 200
+epochLength: 50
 staking:
 slotsPerKESPeriod: 86400
 slotLength: 0.2
 maxKESEvolutions: 90
-securityParam: 10
```

Comment:
Hm, it seemed we used k=10 for shelley and k=8 for byron... Weird, I wouldn't have thought that would work...

Reply:
Why not? `k` is really different between Byron and Shelley on Mainnet as well.

Reply:
> Was  securityParam :param_k: static across eras?

> It is and we (consensus) certainly hope it will remain so

[src](https://input-output-rnd.slack.com/archives/CCZSDE005/p1594041896132800?thread_ts=1594041863.132700&cid=CCZSDE005)

> k is really different between Byron and Shelley on Mainnet as well.

It is not? I guess the "maximum-rollback-length" differs (k vs 3k/f ?), but `k` is the same: 2160.

Reply:
Ho, my bad, I was certain `k` was different by a factor 10.